server/registry/actions_spec_revisions.go

```diff
@@ -93,6 +93,9 @@ func (s *RegistryServer) DeleteApiSpecRevision(ctx context.Context, req *rpc.Del
 	}
 
 	if err := db.DeleteSpecRevision(ctx, name); err != nil {
+		if _, ok := status.FromError(err); ok {
```

Comment:
Could we replace this (and line 99) with a call to grpcErrorFromDBError() ?

Reply:
No, that's a private function to the storage package. But good point: that has already been done by storage, so this doesn't need to translate the error anyway.